tests/headers/CMakeLists.txt

```
@@ -16,7 +16,7 @@ set(exclude_from_headers
   "${PROJECT_SOURCE_DIR}/hpx/lcos/local/futures_factory.hpp"
 )
 
-if(HPX_WITH_PARCELPORT_MPI)
+if(HPX_WITH_NETWORKING AND HPX_WITH_PARCELPORT_MPI)
```

```suggestion
if(HPX_WITH_NETWORKING OR HPX_WITH_PARCELPORT_MPI)
```

but we can also do `if(TARGET Mpi::mpi)` (elsewhere as well) assuming the target is set up early enough.

---

No, NETWORKING should always be enabled if the parcel-port is.

---

FWIW, I think I added the check everywhere.